cwf/gateway/configs/pipelined.yml

```diff
@@ -164,3 +164,5 @@ no_quota_port: 51125
 ###############
 ovs_gtp_port_number: 32768
 ovs_uplink_port_name: cwag_patch
+
+periodic_stats_reporting: True
```

Comment:
Do we want lower case true / false here? Or does yaml take True / False ?

Reply:
Other attributes seem to show capital True/False

Reply:
Actually could you change it to lowercase I think capital case is an outlier we should fix it.

Also lets move this under the `enforcement` block.